slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java

```diff
@@ -4,8 +4,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-@SuppressWarnings({"FinalClassCheck", "HideUtilityClassConstructor"}) class SlidezApplication {
+@SuppressWarnings({ "FinalClassCheck", "HideUtilityClassConstructor" })
+class SlidezApplication {
+
```

Comment:
Don't touch this class, it seems to work now, so please, don't commit it, or Kiril will be angry)

Reply:
In this commit I only ran linter on backend, but I'll remove it